### PR TITLE
chore: update parcel

### PR DESCRIPTION
**Pull request by rtorond**

### CODE REVIEW

File: docs/package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/jefBinomed/SfeirSchoolTheme.git"
+    "url": "git+https://github.com/sfeir-open-source/sfeir-school-theme.git"
```
Comment (JonathanGeoffroy) [inline docs/package.json]: Are you sure you don't want to reference our React school repo instead ?

**Review by JonathanGeoffroy — COMMENTED** [review]
About your profile photo : what about reducing the size and/or compressing it ? 
I do believe git's amazing, but it still 6Mb per photo we'll add for every teacher on every repo thought
Plus : I didn't add a change to test your changes yet, I'll test it ASAP.

File: docs/package.json (hunk 1 above)
Comment (rtorond) [reply docs/package.json]: sure :) done!

**Comment by rtorond** [comment]
> About your profile photo : what about reducing the size and/or compressing it ? I do believe git's amazing, but it still 6Mb per photo we'll add for every teacher on every repo thought Plus : I didn't add a change to test your changes yet, I'll test it ASAP.

Absolutely, didn't notice the size of this picture :p done !

**Review by JonathanGeoffroy — APPROVED** [review]

**Pull request merged**